index.html

```diff
@@ -172,8 +175,25 @@ <h2 class="project-name"> CONTACT  US</h2>
       or a project that needs coding.
       I'd love to help with it
     </p>
-  
-  </div> 
+  <form action="https://formspree.io/f/mgedkakg" method="POST" class="prjct-contact-form">
+    <input name="fullname" id="fullname" class="wrt" maxlength="30" type="text" placeholder=" Enter your name"
+      required>
+    <input name="email" id="email" class="wrt" type="email" placeholder=" Enter your mail" required>
+    <textarea name="msg" class="msg" maxlength="500" placeholder=" Enter your text" required></textarea>
+    <button class="bt" type="submit">see project</button>
+  </form>
+</div> 
+<footer class="finale">
+      <ul class="icones-6">
+        <li><a href="#"><img src="images/icones/sector.png" alt="icon1"></a></li>
+        <li><a href="#"><img src="images/icones/Vecto.png" alt="git icon2"></a></li>
+        <li><a href="#"><img src="images/icones/Vector.png" alt="icon3"></a></li>
+        <li><a href="#"><img src="images/icones/Vector.svg" alt="icon4"></a></li>
+        <li><a href="#"><img src="images/icones/Linkedin.png" alt="linkedinicon"></a></li>
```

Comment:
- [ ] [OPTIONAL] **GOOD JOB 🍀👋🏼🙌🏼🎉💥👏🏼✊🏼.** I have some suggestions that I hope help to improve your design 👍🏼. First add `padding: 12px` to inputs and texture in the form and then you can remove height from just inputs 😉✌🏼. Also, that would be great if you remove the border around the button